zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/tasks/incident/ElasticsearchIncidentUpdateRepository.java

```diff
@@ -75,6 +87,40 @@ public CompletionStage<Map<String, IncidentDocument>> getIncidentDocuments(
         .thenApplyAsync(this::createIncidentDocuments, executor);
   }
 
+  @Override
+  public CompletionStage<Integer> bulkUpdate(final IncidentBulkUpdate bulk) {
+    final var updates = bulk.stream().map(this::createUpdateOperation).toList();
+    final var request =
+        new BulkRequest.Builder()
+            .operations(updates)
+            .source(s -> s.fetch(false))
+            .refresh(Refresh.WaitFor)
```

Comment:
:exclamation: I trust your expertise in settings these parameters, as I don't have much experience in ES queries

Reply:
I discussed it with Svetlana already: https://camunda.slack.com/archives/C06F0GLJNFM/p1732038876206759